packages/account/src/Sections/Security/AccountLimits/account-limits.jsx

```diff
@@ -42,23 +42,29 @@ const Td = ({ children, is_flex }) => (
     </td>
 );
 
-const ExtraInfo = ({ message, ...props }) => (
-    <>
-        <DesktopWrapper>
-            <Popover
-                alignment='top'
-                classNameTargetIcon='account-limit-popover'
-                classNameTarget='account-limit-popover-target'
-                icon='info'
-                message={message}
-                {...props}
-            />
-        </DesktopWrapper>
-        <MobileWrapper>
-            <div className='account-limit-container__extra-info'>{message}</div>
-        </MobileWrapper>
-    </>
-);
+const ExtraInfo = ({ message, ...props }) => {
+    const header_height = 48;
+    const toolbar_height = 56;
```

Comment:
Where are we getting these values from? Are they fixed values from the main `Header` and the `PageOverlay` header?

Reply:
Yes they are.
I've search project to find header and toolbar height in a variable but I didn’t found any variable. So I've added these separately for clarity.
We have same thing on this file:
packages/bot-skeleton/src/scratch/hooks/flyout_base.js

We can export these values from this file and use them in my case and in future.